projects/packages/videopress/src/client/block-editor/blocks/video/edit.tsx

```diff
@@ -44,9 +47,16 @@ import type React from 'react';
 import './editor.scss';
 
 const debug = debugFactory( 'videopress:video:edit' );
-const { myJetpackConnectUrl } = window?.videoPressEditorState || {};
+const { myJetpackConnectUrl, jetpackVideoPressSettingUrl } = window?.videoPressEditorState || {};
 const isUserConnected = getIsUserConnected();
 
+/**
+ * It considers VideoPress active
+ * if the user is connected and the module is active.
+ */
+const isModuleActive = isVideoPressModuleActive();
+const isVideoPressActive = isUserConnected && isModuleActive;
```

Comment:
Do you think it would make sense to move this to the lib as well? Or does `isVideoPressActive` only apply here?

Reply:
yes, I think we can move that to the lib 👍

Reply:
https://github.com/Automattic/jetpack/pull/28722/commits/8b94e97fd6125b3eaec34136159164923dd9da60